cmd/jobsink/main.go

```diff
@@ -312,7 +313,12 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				ReadOnly:  true,
 				MountPath: "/etc/jobsink-event",
 			})
+			mountPathName = "/etc/jobsink-event"
 		}
+		job.Spec.Template.Spec.Containers[i].Env = append(job.Spec.Template.Spec.Containers[i].Env, corev1.EnvVar{
+			Name:  "K_EVENT_PATH",
+			Value: mountPathName,
+		})
```

Comment:
We need to check if the env variable is already defined or not here

Reply:
Actually it was not mentioned, when existing code was checked